parsons/aws/aws_async.py

```diff
@@ -25,21 +25,10 @@ def handler(event, context):
     zappa_run = None
 
 
-def event_command(event, context):
```

Comment:
What's happening with this code change, @rgriff23?

Reply:
The `context` argument is not used in the function, so I deleted it. I can add it back if you know how `context` is supposed to be used.

Reply:
No idea how context is supposed to be used, and you're right that we should remove it if it's not being used. I'm just worried that it could break scripts. If people are passing in a context variable because they don't realize it's not actually used in the function, then they'll get an error when we switch the number of required positional arguments from two to one.

Reply:
Makes sense! I reverted the change.

Reply:
Ok, I'm getting a ton of errors when I try to test this locally. Not sure if that's just me or if it's the PR. For instance, this is the top of the page:

![Screenshot from 2020-10-07 09-59-48](https://user-images.githubusercontent.com/1179362/95341059-e0c9ed80-0883-11eb-971e-6e8c82c756ef.png)

And this is the Redshift quickstart:

![Screenshot from 2020-10-07 09-59-35](https://user-images.githubusercontent.com/1179362/95341087-e7f0fb80-0883-11eb-969e-e54a1e803ea3.png)

In the docs build errors, there's an error:

```
maximum 1 argument(s) allowed, 5 supplied.

.. code-block:: python
  from parsons import Redshift
```
Which suggests that might be a spacing issue?  Anyway, is this just me, or are you also getting docs build errors?

[Here](https://github.com/move-coop/parsons/pull/313#issuecomment-660397875) are instructions for building the docs if you don't have them already.

Reply:
I haven't been building the docs locally, but I am also seeing a lot of errors (and not only in the AWS docs). It'll take me some time to figure all this out.

Reply:
For example, I am also seeing errors from the `build_a_connector` docs, seemingly because it does not like when you end a line with a `:` before a Python code block, e.g.:

```
Literally text:

.. code-block:: python

    print('hello')
```

This has nothing to do with my changes in this PR. Have these errors always appeared, and y'all ignore them when testing the docs locally? Or are they new errors for some reason?

Reply:
I am also seeing warnings unrelated to my PR when I build the docs locally, e.g.:

```
/Users/nunnlab/Documents/GitHub/parsons/docs/build_a_connector.rst:36: WARNING: Could not lex literal_block as "python". Highlighting skipped.
/Users/nunnlab/Documents/GitHub/parsons/docs/build_a_connector.rst:54: WARNING: Could not lex literal_block as "python". Highlighting skipped.
```

Is this expected?

Reply:
I haven't always been testing the docs locally (I don't know about the others) so I think some errors may have snuck through in the past.

Reply:
Not expected, no - when I look at the live docs, I see that those code blocks show up but don't have syntax highlighting, as the errors would lead me to expect. 

I think we should fix all these docs build errors, but you personally don't have to do that. If you can address the issues for this connector, plus any else you feel like working on, that's fine. And we can create an issue to tackle the rest of the errors.

Reply:
There are a few errors in both`SFTP` and `AWS` docs which I've been aware of and had fixed in my 'general fixes' #415 for most of the connectors, but since `SFTP` and `AWS` were unmerged at that time, I failed to include them in my round of cleanups. The issues you highlighted above are examples of these (failure to have a space between `AWS services` and the hyperlink, failure to have a blank line between `.. code:: python` and the first line of code).

The remaining errors seem to be related to my attempts at making hyperlinks to other places in the documentation using labels, which clearly I am doing incorrectly. I'll investigate how to do this correctly.

Reply:
@shaunagm just to update you on the current status, I've fixed all the errors except for this chunk:

```
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:6: WARNING: undefined label: _aws_lambda (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:7: WARNING: undefined label: _aws_s3 (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:8: WARNING: undefined label: _redshift (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:8: WARNING: undefined label: _redshift_table_and_view_api (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:8: WARNING: undefined label: _redshift_schema_api (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:157: WARNING: undefined label: _redshift_table_and_view_api (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/aws.rst:157: WARNING: undefined label: _redshift_schema_api (if the link has no caption the label must precede a section header)
/Users/nunnlab/Documents/GitHub/parsons/docs/databases.rst:104: WARNING: undefined label: _redshift (if the link has no caption the label must precede a section header)
```

I will need to come back to this later, but the good news is there is just one kind of problem that remains to fix now: hyperlinking between different parts of the documentation.

Reply:
That's excellent! Thank you for powering through these, and no worries re: needing to come back later.